<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="OpenGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>641</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="XSlider">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5760</number>
    </property>
    <property name="singleStep">
     <number>16</number>
    </property>
    <property name="pageStep">
     <number>240</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>240</number>
    </property>
   </widget>
   <widget class="QSlider" name="YSlider">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5760</number>
    </property>
    <property name="singleStep">
     <number>16</number>
    </property>
    <property name="pageStep">
     <number>240</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>240</number>
    </property>
   </widget>
   <widget class="QSlider" name="ZSlider">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5760</number>
    </property>
    <property name="singleStep">
     <number>16</number>
    </property>
    <property name="pageStep">
     <number>240</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>240</number>
    </property>
   </widget>
   <widget class="QSlider" name="PropSlider">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>-2</number>
    </property>
    <property name="text">
     <string>X axis slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y axis slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z axis slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Morphing coefficient</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SphereMorphBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Sphere morphing</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FillBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="LineBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DCBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>231</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="StrangeBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Strange morphing</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="NMBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>No morphing</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_fps">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CullingBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>430</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Culling</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="NumbersOfObjectsX">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="NumbersOfObjectsY">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="NumbersOfObjectsZ">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of X axis objecs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Y axis objecs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Z axis objecs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QWidget</extends>
   <header>openglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
